openfe/protocols/openmm_rbfe/equil_rbfe_methods.py

```diff
@@ -41,428 +40,18 @@
-    def is_pressure(cls, v):
-        if not v.is_compatible_with(unit.bar):
-            raise ValueError("Must be pressure value, e.g. use unit.bar")
-        return v
-
-
-class IntegratorSettings(BaseModel):
-    """Settings for the LangevinSplittingDynamicsMove integrator
-
-    Attributes
-    ----------
-    timestep : float * unit.femtosecond
-      Size of the simulation timestep. Default 2 * unit.femtosecond.
-    temperature : float * unit.kelvin
-      Target simulation temperature. Default 298.15 * unit.kelvin.
-    collision_rate : float / unit.picosecond
-      Collision frequency. Default 1 / unit.pisecond.
-    n_steps : int * unit.timestep
-      Number of integration timesteps each time the MCMC move is applied.
-      Default 1000.
-    reassign_velocities : bool
-      If True, velocities are reassigned from the Maxwell-Boltzmann
-      distribution at the beginning of move. Default False.
-    splitting : str
-      Sequence of "R", "V", "O" substeps to be carried out at each
-      timestep. Default "V R O R V".
-    n_restart_attempts : int
-      Number of attempts to restart from Context if there are NaNs in the
-      energies after integration. Default 20.
-    constraint_tolerance : float
-      Tolerance for the constraint solver. Default 1e-6.
-    """
-    class Config:
-        arbitrary_types_allowed = True
-
-    timestep = 2 * unit.femtosecond
-    temperature = 298.15 * unit.kelvin
-    collision_rate = 1.0 / unit.picosecond
-    n_steps = 1000 * unit.timestep
-    reassign_velocities = True
-    splitting = "V R O R V"
-    n_restart_attempts = 20
-    constraint_tolerance = 1e-06
-
-    @validator('timestep', 'temperature', 'collision_rate', 'n_steps',
-               'n_restart_attempts', 'constraint_tolerance')
-    def must_be_positive(cls, v):
-        if v <= 0:
-            errmsg = ("timestep, temperature, collision_rate, n_steps, "
-                      "n_restart_atttempts, constraint_tolerance must be "
-                      "positive")
-            raise ValueError(errmsg)
-        return v
-
-    @validator('temperature')
-    def is_temperature(cls, v):
-        if not v.is_compatible_with(unit.kelvin):
-            raise ValueError("Must be temperature value, e.g. use unit.kelvin")
-        return v
-
-    @validator('timestep')
-    def is_time(cls, v):
-        # these are time units, not simulation steps
-        if not v.is_compatible_with(unit.picosecond):
-            raise ValueError("timestep must be in time units "
-                             "(i.e. picoseconds)")
-        return v
-
-    @validator('collision_rate')
-    def must_be_inverse_time(cls, v):
-        if not v.is_compatible_with(1 / unit.picosecond):
-            raise ValueError("collision_rate must be in inverse time "
-                             "(i.e. 1/picoseconds)")
-        return v
-
-
-class SimulationSettings(BaseModel):
-    """Settings for simulation control, including lengths, writing to disk,
```

Comment:
For tomorrow - I just realised, removing these is probably going to make it impossible to look at the docstring programmattically right? Should we keep them in and then also have the inline docstrings?